src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -828,6 +828,10 @@ private function getPropertyPath($propertyPath): PropertyPath
      */
     public static function createCache($namespace, $defaultLifetime, $version, LoggerInterface $logger = null)
     {
+        if (null === $defaultLifetime) {
```

Comment:
null does not seem to be a valid value at all because if it is passed to ApcuAdapter below, it will crash because the constructor requires int: https://github.com/symfony/symfony/blob/b40f6b5e72459b0854755fec48f2668423ee167d/src/Symfony/Component/Cache/Adapter/ApcuAdapter.php#L23

Reply:
Ah, it's deviding `$defaultLifetime / 5` making it an int again. This type jungling makes me crazy. Good thing we can add types finally. :)

Reply:
This one got me also confused. :-)